aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -115,5 +115,28 @@ Tensor& set_quantizer_(Tensor& self, ConstQuantizerPtr quantizer) {
   return self;
 }
 
+Tensor quantized_clone(const Tensor& self) {
+  TORCH_CHECK(
```

Comment:
do we need this check? afaik we can't create a quantized tensor with non-quantized data type